src/Payment.ts

```diff
@@ -261,23 +261,22 @@ export class Payment {
    * Creates a payment request link.
    * @param networkAddress Address of a currency network.
    * @param amount Requested transfer amount.
-   * @param subject Additional information for payment request.
+   * @param queryParams Additional query params for the generated url
```

Comment:
We should give some documentation about known parameters, like subject, message, ..